examples/aws/rds-s3/README.md

```diff
@@ -0,0 +1,148 @@
+# Kustomize Manifests for RDS and S3
+
+## Overview
+This Kustomize Manifest can be used to deploy Kubeflow Pipelines and Katib with RDS and S3.
+
+Follow the [install](#install) steps below to configure and deploy the Kustomize manifest.
+
+## Provisioning AWS Resources 
+
+### Create S3 Bucket
+
+Run this command to create S3 bucket by changing `<YOUR_S3_BUCKET_NAME>` and `<YOUR_CLUSTER_REGION` to the preferred settings.
+
+```
+export S3_BUCKET=<YOUR_S3_BUCKET_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
+```
+
+### Create RDS Instance
+
+Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance.
+
+## Install
+
+### Install with a single command
+
+Similar to [the single command base installation](../../../README.md#base-install-with-a-single-command) the AWS install configures the Kubeflow official components to integrate with supported AWS services. 
+
+The following sections show how to configure the respective Kubeflow components:
+- [Kubeflow Pipelines](#kubeflow-pipelines-with-rds-and-s3)
+- [Katib](#katib-with-rds)
+
+
+Once configured, install using the following command:
+
+```sh
+while ! kustomize build . | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
```

Comment:
Shall we just keep this step as last one ?

so the way I was thinking 
1. Configure RDS or S3 (Not apply just add the values in `.env` files and all prerequisite )
2. Configure Cognito(Not apply just add the values in `.env` files and all prerequisite )
3. Install kubeflow by running command
   ` while ! kustomize build . | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
`
what do u guys think?

Reply:
Agreed, is there something blocking us from doing this?

another suggestion is to decouple RDS and S3 since some Cx might not want to use long term credentials

Reply:
Actually I've already tried my suggested approach and its there in the quip doc for RDS + S3 and it has worked.